ProcessLib/LiquidFlow/LiquidFlowLocalAssembler.h

```diff
@@ -72,6 +79,9 @@ class LiquidFlowLocalAssembler : public LiquidFlowLocalAssemblerInterface
     using NodalVectorType = typename LocalAssemblerTraits::LocalVector;
     using GlobalDimVectorType = typename ShapeMatricesType::GlobalDimVectorType;
 
+    using LocalMatrixType = Eigen::Map<
+        Eigen::Matrix<double, GlobalDim, Eigen::Dynamic, Eigen::RowMajor>>;
```

Comment:
Just for better understanding for discussion later:
 - Why is one dimension dynamic?
 - Why is it a map and not simply a matrix?

Reply:
As far as I can see this is used for storing a (D x #integration_points) matrix for providing the "input data" of the velocity extrapolation. Therefore:
* Map because the data is stored in a std::vector
* Dynamic: Because of dynamic #int_pts

However, maybe it should be named `VelocityAtIntegrationPointsMatrix` or similar.

Reply:
@chleh Thanks for explanation. The name is changed to MatrixOfVelocityAtIntegrationPoints.